bin/mail/2016-03-21-teacher-recruitment/generate-k5-workshop-attendees

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+attendees = query_subscribed_contacts(q: "kind_s:ProfessionalDevelopmentWorkshopSignup")
+
+export_contacts_to_csv attendees, "k5-workshop-attendees.csv"
```

Comment:
Do we care about naming? These are signups, not attendees. IIRC, there is some measurable difference between these two groups.